Octokit/Clients/RepositorySecretsClient.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    public class RepositorySecretsClient : ApiClient, IRepositorySecretsClient
+    {
+        /// <summary>
+        /// Initializes a new GitHub Repository Branches API client.
+        /// </summary>
+        /// <param name="apiConnection">An API connection</param>
+        public RepositorySecretsClient(IApiConnection apiConnection) : base(apiConnection)
+        {
+        }
+
+        /// <summary>
+        /// Get the public signing key to encrypt secrets for a repository.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/actions/secrets/#get-a-repository-public-key">API documentation</a> for more information.
```

Comment:
I may have missed this earlier, but we need to ensure all the `developer.github.com` links are now our `docs.github.com` since we moved our documentation recently. @mptolly-takeda

Reply:
I'll add a commit after I finish this initial review updating all the `developer.github.com` and `help.github.com` links I find in these changes 👍

Reply:
Yeah, that's something we can do in bulk in a follow-up PR for the existing areas.

For new contributions we should point to the relevant `docs.github.com`  page.